<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerGUIWidgetsModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerGUIWidgetsModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vtkQWidgetWidget test</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Test GUIWidget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="AddHelloWorldGUIWidgetNodeButton">
        <property name="text">
         <string>Add GUI widget node with Hello World push button</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="NewLabelLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="UpdateButtonLabelButton">
          <property name="toolTip">
           <string>To test rendering update on change in widget</string>
          </property>
          <property name="text">
           <string>Update button label</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="AddHomeWidgetButton">
        <property name="text">
         <string>Add GUI widget node with Home widget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddDataModuleWidgetButton">
        <property name="text">
         <string>Add GUI widget node with Data Module widget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddSegmentEditorWidgetButton">
        <property name="text">
         <string>Add GUI widget node with Segment Editor widget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddTransformWidgetButton">
        <property name="text">
         <string>Add GUI widget node with Transform widget</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
